### PR TITLE
CNAM-451: create bulk env and add ssr and had in it

### PR DESCRIPTION
**Pull request by dsun0720**

### CODE REVIEW

File: src/main/resources/flattening/config/bulk/env.conf [hunk 1]
```diff
@@ -0,0 +1,70 @@
...
+    main_table_name = "MCO_B"
+    pmsi_patient_table_name = "MCO_C"
+    tables_to_join = ["MCO_A", "MCO_D", "MCO_UM"]
+  }
+  {
+    name = "MCO_CE"
+    join_keys = ["ETA_NUM", "SEQ_NUM"]
+    main_table_name = "MCO_FASTC"
+    pmsi_patient_table_name = "MCO_CSTC"
+    tables_to_join = ["MCO_FMSTC", "MCO_FBSTC"]
```
Comment (ysebiat) [inline src/main/resources/flattening/config/bulk/env.conf]: You need to add a new table "MCO_FCSTC".

**Review by ysebiat — CHANGES_REQUESTED** [review]
Another table is needed in MCO_CE flattening.

File: src/main/resources/flattening/config/bulk/env.conf (hunk 1 above)
Comment (dsun0720) [reply src/main/resources/flattening/config/bulk/env.conf]: OK

File: src/main/resources/flattening/config/cmap/env.conf [hunk 2]
```diff
@@ -37,6 +37,6 @@ join = [
     name = "MCO_CE"
     join_keys = ["ETA_NUM", "SEQ_NUM"]
     main_table_name = "MCO_CSTC"
-    tables_to_join = ["MCO_FMSTC", "MCO_FASTC", "MCO_FBSTC"]
+    tables_to_join = ["MCO_FMSTC", "MCO_FASTC"]
```
Comment (ysebiat) [inline src/main/resources/flattening/config/cmap/env.conf]: Add  "MCO_FBSTC" to the tables.

**Review by ysebiat — CHANGES_REQUESTED** [review]

File: src/main/resources/flattening/config/cmap/env.conf (hunk 2 above)
Comment (dsun0720) [reply src/main/resources/flattening/config/cmap/env.conf]: do we need MCO_FBSTC in the cmap  ? this one is only used in bulk for extracting ATU event

Comment (ysebiat) [reply src/main/resources/flattening/config/cmap/env.conf]: Yes, we need FBSTC as there is some extractors that use this table.

**Review by ysebiat — DISMISSED** [review]
Approved.

**Review by ysebiat — APPROVED** [review]

**Pull request merged**